From 4cf76bb57d3480420e734da99a5c839ac3a9976a Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Fri, 20 Jan 2017 13:32:06 +0200
Subject: [PATCH] Do not reset the PATH environment variable before running
 scriptlets.

We add lots of native stuff into it and scriptlets rely on that.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

Rebased to 6.0.2
Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
Signed-off-by: Sumanth Gavini<sumanth.gavini@windriver.com>
---
 lib/rpmscript.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/rpmscript.cc b/lib/rpmscript.cc
index b75a62085..2d7a4c889 100644
--- a/lib/rpmscript.cc
+++ b/lib/rpmscript.cc
@@ -260,7 +260,7 @@ static void doScriptExec(ARGV_const_t argv, ARGV_const_t prefixes,
 	if (ipath && ipath[5] != '%')
 	    path = ipath;
 
-	xx = setenv("PATH", path, 1);
+	//xx = setenv("PATH", path, 1);
 	free(ipath);
     }
 
